cetmix_tower_webhook/readme/CONFIGURE.md

```
@@ -0,0 +1,52 @@
+## Configure an Authenticator
+
+- Go to "Cetmix Tower > Settings  > Webhook Authenticators" and click "New".
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)